### PR TITLE
Remove IEncryptionManager

### PR DESCRIPTION
**Pull request by cvium**
**Changes**
Removed IEncryptionManager.

Seems overkill to register a singleton just for de-/encoding from/to Base64. And what triggers me even more is that it's _not_ encryption.

### CODE REVIEW

File: MediaBrowser.MediaEncoding/Subtitles/OpenSubtitleDownloader.cs [hunk 1]
```diff
@@ -63,24 +62,26 @@ void _config_NamedConfigurationUpdating(object sender, ConfigurationUpdateEventA
...
+                options.OpenSubtitlesPasswordHash = EncodePassword(options.OpenSubtitlesPasswordHash);
             }
         }
 
-        private string EncryptPassword(string password)
+        private static string EncodePassword(string password)
         {
-            return PasswordHashPrefix + _encryption.EncryptString(password);
+            var bytes = Encoding.UTF8.GetBytes(password);
+            return PasswordHashPrefix + bytes;
```
Comment (JustAMan) [inline MediaBrowser.MediaEncoding/Subtitles/OpenSubtitleDownloader.cs]: Won't that cause db incompatibility?

Comment (cvium) [reply MediaBrowser.MediaEncoding/Subtitles/OpenSubtitleDownloader.cs]: Copy/paste fail. The OS password is just saved as a base64 encoded string in config/subtitles.xml, but seems I forgot the encoding part.

**Comment by joshuaboniface** [comment]
@Bond-009 review requested - any thoughts on this? If no please merge!

**Comment by cvium** [comment]
Fyi, I noticed that a plugin was using this interface. Based on my research it's not that easy to encrypt files at rest cross platform though. That's why I advocate removing it as it's very misleading.

**Comment by joshuaboniface** [comment]
@cvium Which plugin? If we can fix it up let's do it.

**Comment by cvium** [comment]
@joshuaboniface https://github.com/search?q=org%3Ajellyfin+IEncryptionManager&type=Code

**Pull request merged**